app/components/atoms/NumberHeading/NumberHeading.styles.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+import polygon from './img/Polygon.svg';
+
+export interface NumberHeadingProps {
+  children?: string;
```

Comment:
Remember `children` is reserved for ReactNode type

Reply:
In this case I'm using the component's inner text as a child, should I just set it as a prop or wrap the text in an element to satisfy ReactNode?